src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -20,7 +26,11 @@ public Integer oddSum(List<Integer> numbers) {
      * Вернуть количество вхождений объекта `element`</p>
      **/
     public Long elementCount(List<String> elements, String element) {
-        return null;
+
+        return elements
+                .stream()
+                .filter(x -> (x.toLowerCase().equals(element)))
```

Comment:
we don't need to check ignoreCase, simple equals would be enough here

Reply:
ok